CommentHandler.js

```diff
@@ -0,0 +1,14 @@
+import React from 'react'; 
+
+function CommentHandler(props) {
```

Comment:
It is not a handler, often handler means some event handler, etc. It is a component, but do not need to name it `CommentComponent`, `Comment` will be enough. Please, rename all handlers